docs/index.rst

```diff
@@ -248,6 +248,17 @@ The possible options for each credential profile are:
 Future versions of cbapi will also provide the ability to "pin" the TLS certificate so as to provide certificate
 verification on self-signed or internal CA signed certificates.
 
+Environment Variable Support
```

Comment:
this needs to explicitly state the hierarchy - Environment variables take precedence over a file